ush/wave_grid_interp_sbs.sh

```diff
@@ -133,17 +133,18 @@ source "${USHgfs}/preamble.sh"
 
 # 1.b Run interpolation code
 
+  export pgm="${NET,,}_ww3_gint.x"
+  source prep_step
+
   set +x
-  echo "   Run ww3_gint
-  echo "   Executing ${EXECgfs}/ww3_gint
+  echo "   Executing ${pgm}"
   set_trace
 
-  export pgm=ww3_gint;. prep_step
-  ${EXECgfs}/ww3_gint 1> gint.${grdID}.out 2>&1
+  "${EXECgfs}/${pgm}" 1> gint.${grdID}.out 2>&1
   export err=$?;err_chk
 
 # Write interpolation file to main TEMP dir area if not there yet
-  if [ "wht_OK" = 'no' ]
+  if [ "${wht_OK}" = 'no' ]
```

Comment:
I suspect that this might be the error @WalterKolczynski-NOAA  mentions here https://github.com/NOAA-EMC/global-workflow/pull/3098#issuecomment-2537884401  perhaps the best thing is to back this change out and let's make an issue so it's not holding up this PR?

Reply:
The GEFS job succeeding would make sense because it wouldn't be doing any interpolation so it'd skip this part of the code.   

There might be other issues - but my guess is this is our problem.

Reply:
This is from [PR 307](https://github.com/TerrenceMcGuinness-NOAA/global-workflow/pull/307#issuecomment-2536987316) in my forked repo:

running the C48mx500_hybAOWCDA **FAILED** on Hercules in Build# 6 with error logs:
```
/work2/noaa/global/CI/HERCULES/307/RUNTESTS/COMROOT/C48mx500_hybAOWCDA_22a4e28b/logs/2021032500/gdas_marineanlletkf.log
```
Follow link here to view the contents of the above file(s): [(link)](https://gist.github.com/emcbot/7904cdb8b88240233b60d1993cb6aa03)

All the other CI tests are all languishing in the Priority queues on Hercules

Reply:
I reverted the change and added `FIXME` back to be addressed later.  The script will no longer enter this if-block until the logic is fixed.